ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -249,7 +278,120 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     // P2 -- Usurper Of Frost
+    {
+      id: 'FRU P2 Quadruple Slap',
+      type: 'StartsUsing',
+      netRegex: { id: '9CFF', source: 'Usurper of Frost' },
```

Comment:
The follow-up buster here is `9D00` if we wanted to call both for groups that opt to tank swap instead of cleanse.

Reply:
Hmm. Not sure on the best approach here.  On one hand, doing 3 alerts in a ~8 sec. span (e.g TB1->Swap/Cleanse->TB2) feels pretty noisy.  But a config option also feels a bit heavy....

Reply:
In any case where there's a strat difference, the default callout should include only strat-agnostic information. So the default should just be `Tankbuster` or whatever makes the most sense and is applicable for every strat. If there's need for an additional option due to another strat becoming popular or whatever, a new option can be added then.

Reply:
OK, I think the approach in `3c0de55` & `e8dbaad` is reasonable.  It will call both busters.  But, if the second buster target is the same as the first, the debuff was applied and is still active when the second cast begins (e.g. not negated by Warden's Paean and not yet Esuna'd), and the player can cleanse, it will include a `(Cleanse?)` reminder in the trigger text for the second buster.  I think that's sufficiently strat-agnostic while still conveying the necessary info.